decomp/semantics/uds.py

```diff
@@ -124,19 +140,17 @@ def __init__(self,
                                                      json_name)
                             spl.to_json(json_path)
                             self._graphs.update(spl._graphs)
-
                             self._corpus_paths[sname] = json_path
 
         else:
             self._graphs = graphs
-
             for ann in annotations:
                 self.add_annotation(ann)
 
     @classmethod
     def from_conll(cls,
                    corpus: Union[str, TextIO],
-                   annotations: List[Union[str, TextIO]] = [],
+                   annotations: Dict[Union[str, TextIO], str] = {},
```

Comment:
I don't know. I'm in favor of users explicitly specifying the type of dataset (`raw` or `normalized`), but this could instead be `Dict[str, List[Union[str, TextIO]]]`, where the key is either `raw` or `normalized` and the value is a list of the files or `TextIOs` of that type.